packages/backend/src/graphql/queries/get-users.test.ts

```diff
@@ -0,0 +1,173 @@
+import request, { Test } from 'supertest';
+import app from '../../app';
+import createAuthTokenByUserId from '../../helpers/create-auth-token-by-user-id';
+import createRole from '../../../test/fixtures/role';
+import createPermission from '../../../test/fixtures/permission';
+import createUser from '../../../test/fixtures/user';
+import { IRole, IUser } from '@automatisch/types';
+
+describe('graphQL getUsers query', () => {
+  let query: string;
+
+  beforeEach(async () => {
+    query = `
```

Comment:
as this looks static, we could just move it to line 10 as well without the `beforeEach` block.

Reply:
done 👍